masterlist.yaml

```diff
@@ -25524,19 +25524,36 @@ plugins:
     msg:
       - <<: *patchIncluded
         subs: [ 'Creation Club - Fishing' ]
-        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp") and checksum("SurWR.esp", 538498A4)'
+        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp") and checksum("SurWR.esp", C1861BE4)'
       - <<: *patchUpdateAvailable
         subs: [ '[Rob''s Bug Fixes - Capital Whiterun Expansion](https://www.nexusmods.com/skyrimspecialedition/mods/63355/)' ]
         condition: 'checksum("SurWR.esp", D96CF5BF)'
     clean:
       - crc: 0xD96CF5BF
         util: 'SSEEdit v4.0.4b'
-      - crc: 0x538498A4
+      - crc: 0xC1861BE4
         util: 'SSEEdit v4.0.4b'
   - name: 'SurWR - Fishing.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
         name: 'Rob''s Bug Fixes - Capital Whiterun Expansion'
     clean:
-      - crc: 0x28A22265
+      - crc: 0xA7A3B1AB
         util: 'SSEEdit v4.0.4b'
+
+  - name: 'JKs Skyrim - Rob''s Bug Fixes.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63379/'
+        name: 'Rob''s Bug Fixes - JK''s Skyrim'
+    clean:
+      - crc: 0xE232CE03
+        util: 'SSEEdit v4.0.4b'
+  - name: 'JKs Skyrim - Farming Patch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63379/'
+        name: 'Rob''s Bug Fixes - JK''s Skyrim'
+    clean:
+      - crc: 0x111DD990
+        util: 'SSEEdit v4.0.4b'
+    after:
+      - 'JKs Skyrim - Rob''s Bug Fixes.esp'
```

Comment:
This load after is redundant as the patch matches this regex plugin:

https://github.com/loot/skyrimse/blob/0056c982d8e6c0af8dcd8246f5df52019cb08050/masterlist.yaml#L20137-L20141

```suggestion
```